the-list-ui/src/App.js

```diff
@@ -1,9 +1,27 @@
+import { useEffect, useState } from 'react';
 import './App.css';
 import Input from './components/Input';
+import { getStudents } from './request-functions';
 
 function App() {
+  const [students, setStudents] = useState([{ _id: '', name: '' }]);
+  const [studentsLoadingError, setStudentsLoadingError] = useState();
+
+  const divStudents = [];
+  if (!studentsLoadingError) {
+    for (let i = 0; i < students?.length; i++) {
```

Comment:
Если ты передаёшь массив в качестве значения по-умолчанию в `useState`, то `?` уже не нужен, т.к. `students` не будет `undefined`.

```suggestion
    for (let i = 0; i < students.length; i++) {
```